.travis.yml

```diff
@@ -41,9 +41,9 @@ install:
   # Useful for debugging any issues with conda
   - conda info -a
 
-  - travis_retry conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pytest pandas
+  - travis_retry conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
   - source activate test-environment
-  - travis_retry pip install --only-binary=numpy,scipy numpy nose scipy matplotlib h5py theano
+  - travis_retry pip install --only-binary=numpy,scipy,pandas numpy nose scipy matplotlib h5py theano pytest pytest-pep8 pandas
```

Comment:
Why is the `pytest-pep8` here? Was it installed automatically with conda before?

Reply:
Yes, @gabrieldemarmiesse, The `pytest` in `conda` includes `pytest-pep8` unlike `pip`.